src/toolkit/res_buf.h

```diff
@@ -233,13 +250,13 @@ class ResBuf {
     return r;
   }
 
-  /// Pushes a single resource object to the buffer.
-  /// Resource objects are never combined in the buffer; they are stored as
-  /// unique objects. The resource object is only pushed to the buffer if it
-  /// does not cause the buffer to exceed its capacity.
+  /// Pushes a single resource object to the buffer. If not classified as a bulk
+  /// storage buffer, resource objects are not combined in the buffer; they
+  /// are stored as unique objects. The resource object is only pushed to the
+  /// buffer if it does not cause the buffer to exceed its capacity.
```

Comment:
```suggestion
  /// Pushes a single resource object to the buffer. Unless classified as a 
  // bulk storage buffer, the objects
  /// are stored as unique objects. If classified as a bulk storage buffer, then the 
  // resource objects are combined. The resource object is only pushed to the
  /// buffer if it does not cause the buffer to exceed its capacity.
```
There were quite a few negatives in the same sentence that may get confusing.

Reply:
@gonuke this is from #1687